src/libraries/AssetConfiguration.sol

```diff
@@ -0,0 +1,250 @@
+/**
+    Created by Arcadia Finance
+    https://www.arcadia.finance
+
+    SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity >=0.4.22 <0.9.0;
+
+/**
+ * @title AssetConfiguration library
+ * @author Arcadia Finance
+ * @notice Implements the bitmap logic to handle the asset configurations
+ */
+library AssetConfiguration {
+
+    struct AssetDetailBitmap {
+        // BITMAP
+        //16 bit (0-15)  : Collateral Ratio
+        //16 bit (16-31) : Liquidity Threshold
+        //16 bit (32-47) : Liquidity reward
+        //16 bit (48-63) : Protocol liquidation fee
+        //8  bit (64-71) : Decimals
+        //1  bit (72)    : Asset is active
```

Comment:
Define what active/paused/... is

Reply:
Ill add the details to get set functions natspecs